node/src/stream_handler_pool.rs

```diff
@@ -785,8 +784,9 @@ mod tests {
 
     #[test]
     fn terminal_packet_is_transmitted_and_then_stream_is_shut_down() {
+        use crossbeam_channel::unbounded;
```

Comment:
As I said already. There is no need to use this use clause inside a test; add it to imports above.